README.md

```diff
@@ -128,19 +138,26 @@ Using standard code conventions and infrastructure lowers the barrier for people
   - [x] If web related, submit to [dev.to](https://dev.to)
   - [x] Post on [twitter](https://twitter.com) and upload images of project
   - [x] Post to corresponding reddit programming language community [/r/javascript](https://www.reddit.com/r/javascript), [/r/python](https://www.reddit.com/r/python), etc
+  - [x] When appropriate, consider [talking about your project publicly](https://github.com/vmbrasseur/Public_Speaking)
+    - [x] Review the [Writing and performing a great conference presentation](https://archive.org/details/dodseattle2019-presentation-training) talk
 
 * Specific for Apps / Products
   - [x] Submit project to [Product Hunt](https://www.producthunt.com)
 
-To maintain communication with existing users, use a newsletter. Consider using [mailchimp](https://mailchimp.com) for this.
+To maintain communication with existing users, use a newsletter. Consider using [mailchimp](https://mailchimp.com) for this. Be aware that once communication becomes more fluid, you might create support expectations so it becomes critical to [understand open source support systems](https://nadiaeghbal.com/user-support).
+
+## 🤑 Business model
```

Comment:
I think we should leave the title as `Funding`. I think the term `Business Model` may make people think that we are suggesting they treat their project like a business, which isn't our intent (maybe we can create a different set of advice for people who want to do that).